routes/api.php

```diff
@@ -195,4 +195,14 @@
     'updatePaymentMethod',
 ]);
 
+$router->post('/users/{hash}/reset-key', [
```

Comment:
@rwhite27 what I meant is to change this to /change-email or /update-email , since the method is changeuseremail